sql/src/test/java/io/crate/execution/engine/sort/SortTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.execution.engine.sort;
+
+import com.carrotsearch.randomizedtesting.annotations.Repeat;
+import io.crate.test.integration.CrateUnitTest;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import static org.hamcrest.Matchers.is;
+
+public class SortTest extends CrateUnitTest {
+
+    private ExecutorService executor;
+
+    @Before
+    public void setUpExecutor() throws Exception {
+        executor = Executors.newFixedThreadPool(4);
+    }
+
+    @After
+    public void tearDownExecutor() throws Exception {
+        executor.shutdown();
+        executor.awaitTermination(5, TimeUnit.SECONDS);
+    }
+
+    @Test
+    @Repeat(iterations = 50)
```

Comment:
remove repeat

Reply:
This was kinda intentional given the amount of randomness within the test. It mirrors a bit property based testing where you usually get 100 runs.

Reply:
ah, given the various partitions that might be used... hmm, I see and agree.